qt4/backintime-qt4_polkit

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ $(which pkexec) ]; then
+        pkexec --disable-internal-agent "/usr/bin/backintime-qt4" "$@"
```

Comment:
* No I have a working authentication agent (the built in one from Gnome3).
* Other Applications work just fine.
* Manually starting it in the terminal which `pkexec backintime-qt4` or with ALT F2 run works fine.
* Only starting it over the .desktop file won't work!
* With the mentioned changes it now works like a charm!

About the `--disable-internal-agent` I didn't looked in to it I just changed the calls of GParted in to BIT.